frontend/public/components/routes/create-route.tsx

```diff
@@ -196,11 +227,54 @@ export class CreateRoute extends React.Component<null, CreateRouteState> {
       }));
   }
 
+  newAltServiceEntry() {
+    return {
+      entry: {
+        service: null,
+        weight: 100,
+      },
+      uid: _.uniqueId(),
+    };
+  }
+
+  addAltServiceEntry(){
+    this.setState({
+      alternateServices: _.concat(this.state.alternateServices, this.newAltServiceEntry()),
```

Comment:
This needs to use a function

```js
this.setState(({alernateServices}) => {
  return {alernateServices: [...alernateServices, this.newAltServiceEntry()]};
});
```

see https://reactjs.org/docs/react-component.html#setstate